Frontend/src/pages/admin/AdminCreateNotificationPage.vue

```diff
@@ -0,0 +1,174 @@
+      </div>
+    </q-card>
+  </BasePage>
+</template>
+
+<script>
+import BasePage from "../BasePage.vue";
+
+export default {
+  components: {
+    BasePage
+  },
+  props: {
+    selected: {
+      type: Array,
+      required: false
+    }
+  },
+  name: "HomeMainInfo",
+
+  data: () => ({
+    tableStudents: [],
+    toSendTo: [],
+    body: "",
+    title: "",
+    selectedString: "Please select a student",
+    columns: [
+      {
+        name: "studentLastName",
+        required: true,
+        label: "Student Last Name",
+        align: "left",
+        field: "lastName",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      },
+      {
+        name: "studentFirstName",
+        required: true,
+        label: "Student First Name",
+        align: "left",
+        field: "firstName",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      },
+      {
+        name: "studentID",
+        required: true,
+        label: "Student ID",
+        align: "left",
+        field: "id",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      },
+      {
+        name: "email",
+        required: true,
+        label: "Email",
+        align: "left",
+        field: "email",
+        sortable: true,
+        classes: "my-class",
+        style: "width: 500px"
+      }
+    ]
+  }),
+  created: function() {
+    this.$axios
+      .get("/students", {
+        headers: {
+          Authorization: this.$store.state.token
+        }
+      })
+      .then(resp => {
+        this.tableStudents = resp.data;
+      });
+    this.toSendTo = this.selected;
+  },
+  methods: {
+    goToStudentCoop() {
+      this.$router.push("/admin/studentcoops");
+    },
+    sendNotification() {
+      var adminId = this.$store.state.currentUser.id;
+      var stuIds = [];
+      for (var i = 0; i < this.toSendTo.length; i++) {
+        stuIds.push(this.toSendTo[i].id);
+      }
+
+      // create the notification
+      this.$axios
+        .post(
+          "/notifications/many?admin=" +
+            adminId +
+            "&title=" +
+            this.title +
+            "&body=" +
+            this.body +
+            "&studentIds=" +
+            stuIds
+        )
+        .then(_resp => {
+          this.body = "";
+          this.title = "";
+          this.toSendTo = [];
+          this.$q.notify({
+            color: "green-4",
+            position: "top",
+            textColor: "white",
+            icon: "cloud_done",
+            message: "Notification(s) sent successfully!"
+          });
+        })
+        .catch(_err => {
+          this.$q.notify({
+            color: "red-4",
+            position: "top",
+            textColor: "white",
+            icon: "error",
+            message: "Something went wrong, please try again"
+          });
+        });
+    }
+  }
+};
+</script>
+
+<style scoped lang="scss">
+h6 {
+  margin: 10px;
```

Comment:
You can use the `q-ma-md` class instead of defining the styles yourself (`ma` = margin-all, `md`= medium). https://quasar.dev/style/spacing#Examples

Reply:
this literally never ends up working the way i want it to so i really dont want to waste more time trying to get this shit to align properly so i dont rlly want to go down this road again

Reply:
Ok no worries it's not a big deal